drawBot/context/baseContext.py

```diff
@@ -1391,7 +1391,7 @@ def openTypeFeatures(self, *args, **features):
             # create an empty formatted string object
             t = FormattedString()
             # set a font
-            t.font("ACaslonPro-Regular")
+            t.font("Didot")
```

Comment:
But this font does not have 'lnum'. The example should switch to 'onum' I think.